### ISSUE TITLE
Internal Error: -62 (KErrNotReady)

### ISSUE THREAD

**Issue by jbmorley** [inseven/opolua#223]
## Description

Running Clock5. 

## Details

```
-62 (KErrNotReady)
CLOCK5\INITIALISE:0000FA5B: FF [NextOpcodeTable] 18 CallOpxFunc 0 43 (system LoadRsc)
CLOCK5\CLOCK5:0000066E: 53 [RunProcedure] 0x0579 (name="INITIALISE" nargs=0)
stack traceback:
	[C]: in function 'assert'
	opl.lua:1148: in function <opl.lua:1145>
	(...tail calls...)
	system.lua:359: in function 'opx.system.LoadRsc'
	ops.lua:2327: in function 'ops.CallOpxFunc'
	ops.lua:2209: in function 'ops.NextOpcodeTable'
	runtime.lua:816: in function <runtime.lua:801>
	[C]: in function 'xpcall'
	runtime.lua:933: in method 'pcallProc'
	runtime.lua:1288: in function 'runtime.runOpo'
```

**Comment by tomsci** [comment]
This one is a simple bug in LoadRsc not calling `abs()` on the path, but there are several other issues with Clock5 hiding behind it...

**Comment by tomsci** [comment]
Fixed in 887fca8a89d36faf890349011a7fd6693e08206c (and many subsequent commits for other issues found with Clock5...)
